clang/test/CodeGenSYCL/num-simd-work-items.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clang_cc1 -std=c++11 -triple spir64-unknown-linux-sycldevice -disable-llvm-passes -fsycl-is-device -emit-llvm -o - %s | FileCheck %s
```

Comment:
```suggestion
// RUN: %clang_cc1 -std=c++11 -triple spir64-unknown-unknown-sycldevice -disable-llvm-passes -fsycl-is-device -emit-llvm -o - %s | FileCheck %s
```

Reply:
Oops, I thought that I fixed it somewhere already. Lets try github feature to commit in review.

Reply:
Ah, it's working, but not in the way our testing accepts.